src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const i in robot) {
```

Comment:
What means "obj"? Also, "i" is for iterator, here it is key

Reply:
"obj" means "object'" from the task _"...and returns a new object in which..."_ but I agree "new Robot" is more understandable) Thanks for "i"!